dev-python/mdx_gh_links/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!--maintainer-needed-->
```

Comment:
We can't add a new package to tree without dedicated maintainer (I hope this is just some overlap when bringing package from guru...)
https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602

Reply:
Oopsie, I though I had already added myself to the metadata files, but apparently not, should be fixed now